sql/core/src/test/resources/sql-tests/results/show-tables.sql.out

```diff
@@ -117,6 +117,63 @@ show_t2
 show_t3
 
 
+-- !query
+CACHE TABLE show_t1
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+CACHE TABLE show_t5 AS SELECT 1
+-- !query schema
+struct<>
+-- !query output
+
+
+
+-- !query
+SHOW CACHED TABLES LIKE 'show_t*'
+-- !query schema
+struct<database:string,tableName:string,isTemporary:boolean>
+-- !query output
+show_t1
+show_t5
```

Comment:
nit: hm, `SQLQueryTestSuite` only outputs table names for the show command, so it might be better to add some tests somewhere, too.

Reply:
added test in `DDLSuite`